### ISSUE TITLE
Missing Test Case - 1084. Sales Analysis III

### ISSUE THREAD

**Issue by kathija-rasheed** [LeetCode-Feedback/LeetCode-Feedback#25985]
### LeetCode Username

kathija-rasheed

### Problem Number, Title, and Link

1084. Sales Analysis III https://leetcode.com/problems/sales-analysis-iii/

### Bug Category

Missing test case *(Incorrect/Inefficient Code getting accepted because of missing test cases)*

### Bug Description

Query considers only if it was sold in first quarter, in all years.(not 2019 alone). Problem statement clearly states from '2019-01-01' to '2019-03-31' . 

### Language Used for Code

MySQL

### Code used for Submit/Run operation

```bash
SELECT P.product_id, P.product_name FROM Product P 
RIGHT JOIN(
    SELECT product_id, EXTRACT(QUARTER FROM sale_date) AS s FROM Sales
    GROUP BY product_id
    HAVING COUNT(DISTINCT s) = 1 AND s = 1) AS Q
    ON P.product_id = Q.product_id;
```

### Expected behavior

Above query considers only if it was sold in first quarter, in all years.(not 2019 alone). Problem statement clearly states from '2019-01-01' to '2019-03-31' . I think the sales table contains data about 2019 only, 
So basically solution should not have been accepted.

**Comment by kathija-rasheed** [comment]
its more about missing data in tables. If the table contains sale_date as '2020-01-01' my query wont be accepted, but right now it does get accepted
